table/block.cc

```diff
@@ -140,6 +140,31 @@ void BlockIter::Prev() {
 }
 
 void BlockIter::Seek(const Slice& target) {
+  Slice seek_key = target;
+  if (!key_includes_seq_) {
```

Comment:
Why do we still need this?

Reply:
The function is shared between BlockIter and IndexBlockIter. If we make two copies of the function then we might cause more code cache misses. Relying on cpu branch prediction is a better bet.